### ISSUE TITLE
Channel and normalize settings are not applied to cut audio

### ISSUE THREAD

**Issue by JJKylee** [staxrip/staxrip#713]
### Describe the bug

StaxRip version: 2.6.0

Project Options ⇒ Demux Audio: `None`

When the `Cutting` filter is applied, the channel setting in audio does not apply to the cut audio. And `normalize` also does not apply to the cut audio.

This is true regardless of the _Decoding Method_ setting in audio when the source is an _mkv_ file, because the audio stream is _always_ piped in that case.

### Expected behavior

The channel setting and normalize should be correctly applied.

### How to reproduce the issue

The source file is [here](https://drive.google.com/file/d/1932O54x9mWtqhQMo0FypcCqWGltX7uKH/view?usp=sharing)(113 MiB, MBAFF interlaced 422 10bit VFR).

Encode the source file with settings as follows:

![StaxRip_2 6 0_cut_audio_pipe_channel_setting_issue_1_20210522](https://user-images.githubusercontent.com/3157518/119243145-0ce76580-bb19-11eb-9359-aafdefe26a20.png)

![StaxRip_2 6 0_cut_audio_pipe_channel_setting_issue_2_20210522](https://user-images.githubusercontent.com/3157518/119243148-1375dd00-bb19-11eb-8311-034ef004496e.png)

![StaxRip_2 6 0_cut_audio_pipe_channel_setting_issue_3_20210522](https://user-images.githubusercontent.com/3157518/119243149-1b358180-bb19-11eb-88b8-f3beb0e2d615.png)

Although the audio channel is set to _Stereo_, the output audio has the same channels (5.1) as the source. And `normalize` works on audio tracks as a whole, not on the cut audio. In addition, all audio tracks are re-encoded unnecessarily although it was not set in the project.

### The cause

Here is the log file:
[MBAFF interlaced 422 10bit VFR sample_new_staxrip.log](https://github.com/staxrip/staxrip/files/6527298/MBAFF.interlaced.422.10bit.VFR.sample_new_staxrip.log)

We can see that in line 873, the cut audio is piped to `qaac` _without the proper channel setting_, `-ac 2`.

And as you can see in the log, the `--normalize` option is also _not_ applied to the _cut_ audio. Instead, `volumedetect` and `normalize` are applied to the source audio tracks and they are re-encoded unnecessarily regardless of the project settings.

### Extra info

[Log Viewer Bug?]

I've found another bug related to the _log viewer_. Some parts have _the same header_, and thus clicking the header always leads to the _**first**_ corresponding location in the log, not the actual correct place.

For example, _Audio Encoding 1_ appears three times in the log provided above, and when viewed in the log viewer, clicking _Audio Encoding 1_ always jumps to the _first_ occurence in the log, not the correct one.

**Comment by Dendraspis** [comment]
Confirmed,  but there is even more, because it also happens without cutting!

**Comment by 44vince44** [comment]
Not a bug, imho.

There are kinds of processing that must be done with ffmpeg, and ffmpeg needs to works on the demuxed stream (this is as er Staxrip architecture)

If you want to process this kind of things, you need to DEMUX audio.

**Comment by JJKylee** [comment]
It's _**definitely a bug**_. Demux doesn't matter.
Here's the log when I demux all audio tracks and do the same cutting with the same audio encoding options.

[MBAFF interlaced 422 10bit VFR sample_new_staxrip.log](https://github.com/staxrip/staxrip/files/6528555/MBAFF.interlaced.422.10bit.VFR.sample_new_staxrip.log)

This time, unnecessary re-encoding of the whole audio tracks are not done, but still `Stereo` and `normalize` don't work.

Look at lines 580 and 757.

```
Channel(s)               : 6 channels
```

**Comment by Patman86** [comment]
you're right @44vince44 , i remember a similar discussion. @stax76 then had clarified that eight times, i think.

**Comment by 44vince44** [comment]
@JJKylee I'll try to reproduce it with your own file later (i hope)... I'm being busy lately

**Comment by JJKylee** [comment]
Guys,

Sorry but it seems like there must have been something wrong with my StaxRip.

I fresh installed another copy of StaxRip and did the same testing with it. And now `normalize` and `Stereo` work OK with and without demux, and regardless of whether the audio is piped or not.

Geez...

I don't know what went wrong along the way. 😓

Sorry about the fuss again. 🙏

**Comment by 44vince44** [comment]
Well that's nice of you to tell us this REASSURING news, even though you did just after I finished my test with your source, in which I made a wonderful video with two STEREO tracks.
SO all OK, and confirmed it works!

**Comment by JJKylee** [comment]
I've been updating `Settings.dat` by updating all the settings/profiles/templates every time I installed a new version, but maybe it was not enough. There must have been something missing in the course, and I can't figure out what it was. Geez... 😓

**Comment by Dendraspis** [comment]
@JJKylee 
What matters is that it works. 🤗

**Comment by JJKylee** [comment]
After spending 5 hours to figure out what caused the problem, I think I've found out why. It was the _Frameserver_ setting.

I've been fixing it at _Use installed directly_ for a long time after deleting the portable `frameserver` folder in the original distribution.

This time, I fresh installed StaxRip, but removed the `frameserver` folder, put `ffmpeg.exe` in a separate `ffmpeg` folder, set the frameserver to `installed`, and did all the settings to my taste. But I still couldn't get the stereo audio.

Looking into/comparing the logs closely, I've found out the only difference: `ffmpeg.exe` in the portable AviSynth folder worked OK, but my `ffmpeg.exe` in a separate folder did not.

So I **reset the _Frameserver_ setting** by setting it to back to `portable` and then to `installed`. (`frameserver` folder was not necessary.) Now it works as expected.

I think this was the missing loophole in updating the settings/profiles/templates after the release. 😭
